it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartControllerISpec.scala

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.controllers
+
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentType, _}
+import uk.gov.hmrc.agentsubscriptionfrontend.stubs.AuthStub
+import uk.gov.hmrc.agentsubscriptionfrontend.support.SampleUsers._
+
+class StartControllerISpec extends BaseControllerISpec {
+
+  private lazy val controller: StartController = app.injector.instanceOf[StartController]
+
+  "context root" should {
+    "redirect to start page" in {
+      val result = await(controller.root(FakeRequest()))
+
+      status(result) shouldBe 303
+      redirectLocation(result).head should include ("/start")
+    }
+  }
+
+  "start" should {
+    "not require authentication" in {
+      AuthStub.userIsNotAuthenticated()
+
+      val result = await(controller.start(FakeRequest()))
+
+      status(result) shouldBe 200
+    }
+
+    "be available at /agent-subscription/start" in {
```

Comment:
We did originally start using `route()` but were unable to pass session keys into it.

Since then I came accross something that might allow us to pass session keys when using `route()` so if you're interested we can look into that.

Not directly relevant to this test except that it would be a bit inconsistent to use `route()` here but not in the tests for authenticated endpoints that need session keys.